### PR TITLE
Adding Information Prerequisites Code of Conduct

### PR DESCRIPTION
**Pull request by Radhika-Saini**
I made the changes in the Code of Conduct section.

### CODE REVIEW

**Comment by Radhika-Saini** [comment]
@ssurbhi560 @kathatherine @beckermr Kindly review.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Thanks! Review done. For future reference, you don't need to close your PRs after being given a review. Please just make the changes and push the new commit to the same branch. Your changes will show up in the PR.

**Comment by Radhika-Saini** [comment]
@kathatherine  Thanks for the feedback.
I made the changes. Kindly review commit-2.

File: src/README.md [hunk 1]
```diff
@@ -13,3 +13,85 @@ Documentation update PRs should only change the files in ``src``.
 
 Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
 
+
+# Code of conduct
+
+Conda-forge follows the NumFOCUS Code of Conduct-
```
Comment (kathatherine) [inline src/README.md:19]: Let's do a colon (:) instead of a hyphen (-), please.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Just that one change and you should be good!

**Review by kathatherine — APPROVED** [review]
Looks good. Thanks!

**Comment by jaimergp** [comment]
Superseded by #1640

**Pull request closed without merging**